dbcrossbarlib/src/drivers/postgres/local_data.rs

```diff
@@ -5,13 +5,13 @@ use failure::Fail;
 
 use super::{connect, PostgresLocator};
 use crate::common::*;
-use crate::drivers::postgres_shared::{CheckCatalog, PgCreateTable};
+use crate::drivers::postgres_shared::{CheckCatalog, PgCreateTable, TableName};
 
 /// Copy the specified table from the database, returning a `CsvStream`.
 pub(crate) async fn local_data_helper(
     ctx: Context,
     url: UrlWithHiddenPassword,
-    table_name: String,
+    table_name: TableName,
```

Comment:
nice

Reply:
Yeah, this makes big difference in terms of code clarity and correctness.